packages/block-library/src/spacer/controls.js

```diff
@@ -51,7 +52,15 @@ function DimensionInput( { label, onChange, isResizing, value = '' } ) {
 		}
 	};
 
-	const inputValue = temporaryInput !== null ? temporaryInput : value;
+	const computedValue = useMemo( () => {
```

Comment:
Similar feedback to the Cover block PR — do we need the `temporaryUnit` internal state? Over the course of changes in this component, it currently looks like `temporaryInput` is always `null`. Is it worth us removing it at this stage?

Reply:
Good point, I went ahead and remove it